src/main.rs

```diff
@@ -808,6 +901,16 @@ fn main() {
         return;
     }
 
+    if let Err(e) = expand_fdtable() {
+        match e {
+            FdTableError::Close(_) => {
+                eprintln!("Error expanding FD table: {e}");
+                std::process::exit(1);
```

Comment:
Since expand_fdtable was introduced because "it's a best-effort solution to the delays caused by RCU synchronization", so here should not exit the whole process once expand fails.

Reply:
Ok, removed that part.